solr/solr-ref-guide/modules/upgrade-notes/pages/major-changes-in-solr-10.adoc

```diff
@@ -26,6 +26,10 @@ Before starting an upgrade to this version of Solr, please take the time to revi
 
 // TODO add similar text that previous releases have at this spot.
 
+== Required Java Version Changes
```

Comment:
Let's use general headings that aren't super specific to the actual issue, so that other things might also be added here.  So maybe rename to "System Requirements"